js/src/components/legacy-plugin-active-notice/index.scss

```diff
@@ -0,0 +1,7 @@
+.sfw-legacy-plugin-active-notice {
```

Comment:
@iamdharmesh Can we wrap the `LegacyPluginActiveNotice` within the `Section` component so that we avoid having those duplicate CSS to center the notice? We'll have to update the `Section` component to not render `.sfw-section__header` if there's no `title`, `description` and `topContent`.
Also, why are we overwriting the font size?